src/client/styles/base.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+@import "colours";
+
+@import "resets";
+@import "footer";
+@import "form";
+@import "header";
+@import "spinner";
```

Comment:
Much cleaner than in the Udacity example. Nice!

Reply:
Thank you. You showed me 😛